problemsets-mat223/modules/module11.tex

```diff
@@ -376,10 +376,14 @@
 trying to be extra precise, we must make sure our boxes/lists of numbers and our transformations/vectors stay separate.
 
 \begin{example}
-	Let $\mathcal T$ be the transformation induced by the matrix $M=\mat{1&2&5\\2&-2&-2}$,
+	Let $\mathcal T$ be the transformation induced by the matrix $M=[\mathcal T_{M}\vec v]_{\mathcal E'}= M[\vec v]_{\mathcal E}$,
```

Comment:
No. What you've written is incorrect. Put it back the way it was.

Reply:
Yeah, I have no idea why I change the question